cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -1189,7 +1191,7 @@ func (csr *ClusterStateRegistry) handleInstanceCreationErrorsForNodeGroup(
 				ErrorClass:   errorCode.class,
 				ErrorCode:    errorCode.code,
 				ErrorMessage: csr.buildErrorMessageEventString(currentUniqueErrorMessagesForErrorCode[errorCode]),
-			}, gpuResource, gpuType, currentTime)
+			}, gpuResource, gpuType, currentTime, errorCode.code != "timeout")
```

Comment:
i'd like to see `"timeout"` replaced with a constant value.

also, this reference as well, https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/clusterstate/clusterstate.go#L307

could you create a constant value that lives in `clusterstate/clusterstate.go` and then use the constant value for this check and also the error code creation. i think this will make the code less brittle.